src/encoded/schemas/mouse_donor.json

```diff
@@ -1,47 +1,35 @@
 {
     "title": "Mouse donor",
-    "description": "Schema for submitting a mouse donor.",
+    "description": "Schema for submitting a mouse strain.",
+    "comment": "The mouse donor object is meant to represent a strain not an individual mouse.",
     "id": "/profiles/mouse_donor.json",
     "$schema": "http://json-schema.org/draft-04/schema#",
     "type": "object",
     "required": [ "award", "lab", "organism"],
     "identifyingProperties": ["uuid", "accession", "aliases"],
     "additionalProperties": false,
     "mixinProperties": [
-        { "$ref": "mixins.json#/schema_version" },
-        { "$ref": "mixins.json#/uuid" },
         { "$ref": "donor.json#/properties" },
-        { "$ref": "mixins.json#/accession" },
-        { "$ref": "mixins.json#/aliases" },
-        { "$ref": "mixins.json#/attribution" },
-        { "$ref": "mixins.json#/accessioned_status" },
+        { "$ref": "mixins.json#/genetic_modifications" },
+        { "$ref": "mixins.json#/strains"},
         { "$ref": "mixins.json#/source" },
-        { "$ref": "mixins.json#/submitted" },
         { "$ref": "mixins.json#/references" },
-        { "$ref": "mixins.json#/notes" },
         { "$ref": "mixins.json#/documents" },
-        { "$ref": "mixins.json#/strains"},
-        { "$ref": "mixins.json#/genetic_modifications" }
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/accessioned_status" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/schema_version" }
     ],
     "properties": {
         "schema_version": {
-            "default": "9"
-        },
-        "littermates": {
-            "title": "Littermates",
-            "description": "Donors comprising the same litter.",
-            "type":  "array",
-            "default": [],
-            "uniqueItems": true,
-            "items": {
-                "title": "Littermate",
-                "description": "A donor member of the same litter.",
-                "comment": "See mouse_donor.json for available identifiers.",
-                "type": "string",
-                "linkTo": "MouseDonor"
-            }
+            "default": "10"
         },
         "organism": {
```

Comment:
Wow I did not know that is how we hard wired MouseDonor to Mouse taxon id.

Reply:
it is hard wired in human and mouse and restricted in fly and worm (I assume to prevent unintended mistakes)